client/client.go

```diff
@@ -0,0 +1,69 @@
+// Package httpclient provides a testing enviroment to test poki API server.
+package httpclient
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+type Client struct {
+	httpClient *http.Client
+	URL        string
+	endPoint   string
+}
+
+type Option func(c *Client)
+
+type Pokemon struct {
+	ID             int    `json:"id"`
+	Name           string `json:"name"`
+	BaseExperience int    `json:"base_experience"`
+	Height         int    `json:"height"`
+}
+
+type BodyFormat struct {
+	Pokemons []Pokemon `json:"results"`
+}
+
+// NewClient creates a new client
+func NewClient(options ...Option) *Client {
+
+	err := godotenv.Load("../.env")
```

Comment:
client shouldn't handle reading envvars it should be read in the main and passed to the factory method